static/babelrc.json

```diff
@@ -1,7 +1,12 @@
 {
-  "breakConfig": true,
   "sourceMap": "inline",
-  "blacklist": ["es6.forOf", "useStrict"],
-  "optional": ["asyncToGenerator"],
-  "stage": 0
+  "plugins": [
+    "add-module-exports",
+    "transform-async-to-generator",
+    "transform-class-properties",
+    "transform-es2015-modules-commonjs",
```

Comment:
I seem to remember that there was *some* kind of issue with this transform and strict mode... but I can't for the life of me remember what it was. -_-